contracts/utils/ICS02ClientUpgradeable.sol

```diff
@@ -83,19 +86,49 @@ abstract contract ICS02ClientUpgradeable is IICS02Client, IICS02ClientErrors, Ac
     )
         external
         returns (string memory)
+    {
+        string memory clientId = nextClientId();
+        _addClient(clientId, counterpartyInfo, client);
+        return clientId;
+    }
+
+    /// @inheritdoc IICS02Client
+    function addClient(
+        string memory clientId,
+        IICS02ClientMsgs.CounterpartyInfo calldata counterpartyInfo,
+        address client
+    )
+        external
+        onlyRole(CLIENT_ID_CUSTOMIZER_ROLE)
+        returns (string memory)
+    {
+        require(bytes(clientId).length != 0, IBCInvalidClientId(clientId));
```

Comment:
Should‘t we also check that the custom name is not in use? Because we'll just overwrite it here otherwise.

Reply:
Yep, added the check